Resources/Prototypes/_Andromeda/Entities/Objects/Guns/Pistol/pistols.yml

```
@@ -0,0 +1,50 @@
+        - Burst 
+      soundGunshot:
+        path: /Audio/Weapons/Guns/Gunshots/mk58.ogg
+      fireOnDropChance: 0.5
+
+- type: entity
+  name: mx3r
+  parent: WeaponPistolMX3R
+  id: WeaponPistolMX3REmpty
+  suffix: Empty
+  components:
+  - type: ItemSlots
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)